app/controllers/ctc/questions/questions_controller.rb

```diff
@@ -7,6 +7,10 @@ class QuestionsController < ::Questions::QuestionsController
 
       private
 
+      def set_ctc_year
```

Comment:
what is this?

Reply:
This was an attempt to make all CTC questions page have a `@current_tax_year`, but then we noticed we didn't strictly need that, so we lived without. We should toss this line, then think about what we do want to add, then add that.